s3api/server_test.go

```diff
@@ -39,9 +39,9 @@ func TestNew(t *testing.T) {
 	port := ":7070"
 
 	tests := []struct {
-		name            string
-		args            args
 		wantS3ApiServer *S3ApiServer
+		args            args
+		name            string
```

Comment:
i usually see test name as first member fro readability in tests, seems thats better than the size optimization here?